logstreams/src/main/java/io/zeebe/logstreams/impl/log/LogStreamImpl.java

```diff
@@ -284,7 +284,7 @@ private void internalSetCommitPosition(final long commitPosition) {
               if (throwable == null) {
                 appender =
                     new LogStorageAppender(
-                        buildActorName(nodeId, "LogAppender-" + partitionId),
+                        buildActorName(nodeId, "LogAppender", partitionId),
```

Comment:
Can also update the Dispatcher actor name. A few lines above - L273. Not sure how to comment there.